interop-testing/src/main/java/io/grpc/testing/integration/TestServiceServer.java

```diff
@@ -34,9 +34,14 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-/** Server that manages startup/shutdown of a single {@code TestService}. */
```

Comment:
Why were these reformatted in the recent commit? Are you using full-file reformatting? If so, please don't, because unless everyone uses the same identical version, they fight and produce garbage conflicts. This PR seems to have a lot of unnecessary reformatting, so I assume you frequently used full-file reformatting.

Only reformat lines you changed and, in a mixed PR, sometimes lines that were improperly formatted. "Improperly" meaning "against the style guide," not "different from how I would format it." So that's like "the indentation level was wrong," and not "adjust this line that was wrapped at 80 columns instead of 100."

Reply:
Apologies, I was using full-file reformatting before whenever I changed a file.
I have reverted all the unnecessary reformatting in the latest commit and only reformat the lines I have changed.
Will keep this in mind the future.